src/protocol/sort/shuffle.rs

```diff
@@ -0,0 +1,99 @@
+use crate::helpers::prss::PrssSpace;
+
+#[allow(dead_code)]
+
+pub struct Shuffle {}
```

Comment:
do we expect state to be added later or we can just replace this struct by a single function that implements FY shuffle? btw, I think Rust std does have an implementation of FY shuffle

Reply:
Great to know.. I will use the Rust std library instead. Yes this struct will be filled up in the next diff